assets/src/styles/partials/_tooltip.scss

```diff
@@ -1,63 +1,23 @@
-// Based off http://refills.bourbon.io/components/
-.tooltip-item {
-  $base-border-color: gainsboro !default;
-  $base-border-radius: 3px !default;
-  $base-line-height: 1.5em !default;
-  $dark-gray: #333 !default;
-  $base-font-color: $dark-gray !default;
-  $tooltip-background: white;
-  $tooltip-color: $base-font-color;
-  $tooltip-width: 16em;
-  $tooltip-shadow: 0 2px 2px silver;
-  $tooltip-distance-from-item: 1em;
+// Modified CSS for USWDS Tooltip component
+.usa-tooltip__body {
+    width: auto;
+    white-space: pre-wrap;
+    width: 40vw;
+	max-width: 450px;
 
-  background: $tooltip-background;
-  border-radius: $base-border-radius;
-  display: inline-block;
-  position: relative;
-  text-align: center;
-
-  // On pointer devices, use the focus and hover state to indicate visibility. Can't use hover media query as it is
-  // not implemented on Firefox or IE11.
-  &:focus,
-  &:hover .tooltip {
-    opacity: 1;
-    visibility: visible;
-  }
-
-  // On touch devices, use the show-tooltip class to indicate visibility.
-  // See helpers.js for corresponding Javascript.
-  @media (hover: none) {
-    &.show-tooltip .tooltip {
-      opacity: 1;
-      visibility: visible;
-    }
-  }
+	@media only screen and (max-width: 700px) {
+		width: 40vw;
+		max-width: 250px;
+		font-size: 14px;
+	}
+}
 
-  .tooltip {
-    word-wrap: normal;
-    position: absolute;
-    left: $tooltip-distance-from-item;
-    background-color: $tooltip-background;
-    background: #fff;
-    border-radius: $base-border-radius;
-    box-shadow: $tooltip-shadow;
-    color: $tooltip-color;
-    font-size: 0.9em; // Make sure you use -webkit-backface-visibility: hidden; on the body element to prevent 1px nudging bugs.
-    line-height: 1.5em;
-    margin: 0 auto;
-    width: $tooltip-width;
-    opacity: 0;
-    padding: 1em;
-    text-align: center;
-    transition: all 0.2s ease-in-out;
-    visibility: hidden;
-    z-index: 1000;
+// Rendered USWDS tooltips that aren't initialized by USWDS
+.render-tooltip:hover{
```

Comment:
I'm hoping that the work Blake is doing will allow us to not have to put this in here.